cromwell_tools/cromwell_tools.py

```diff
@@ -109,24 +116,29 @@ def wait_until_workflow_completes(
 
 def start_workflow(
         wdl_file, inputs_file, url, options_file=None, inputs_file2=None, zip_file=None, user=None,
-        password=None, label=None):
+        password=None, label=None, validate_inputs=True):
```

Comment:
I would call this validate_labels, since it doesn't validate the inputs json.

Reply:
Will do. I called it `validate_inputs` here previously because I imagined we would have more validators(wdl file validators, inputs validators) in the future in the cromwell-tools. But now I think for clarity it is better to call the flag `validate_labels`.